app/hotels/src/navigation/AllHotelsMapNavigationScreen.js

```diff
@@ -16,6 +17,14 @@ type Props = {|
 export default class AllHotelsMapNavigationScreen extends React.Component<
   Props,
 > {
+  static navigationOptions = ({ navigation }: Props) => {
+    function goBack() {
+      navigation.goBack(null);
+    }
+    return {
+      headerLeft: <HeaderBackButton tintColor="#fff" onPress={goBack} />,
+    };
+  };
```

Comment:
@tbergq I don't understand our navigation system. Do we have to call this on every screen?

Reply:
Not on every screen, but on the first level of each nested stack yes.

Reply:
That's silly. So why is there `StackNavigator` only for one map screen?